src/components/cell-tooltip/CellTooltipText.js

```diff
@@ -10,14 +10,16 @@ export default function CellTooltipText(props) {
   } = props;
 
   const ref = useRef(null);
+  const reduceSize = (parentWidth < 300);
```

Comment:
Slight preference for naming variables based on what they measure, rather than their intended use.